smsdk/client.py

```diff
@@ -48,7 +48,9 @@ def dict_to_df(data, normalize=True):
                 # machine type stats are list
                 cols = [*data[0]]
                 cols.remove("stats")
-                df = json_normalize(data, "stats", cols, record_prefix="stats.")
+                df = json_normalize(
+                    data, "stats", cols, record_prefix="stats.", errors="ignore"
```

Comment:
Why are we ignoring errors here?

Reply:
While getting get_machine_types() with v1 endpoint we are getting `part_types` in response but not always that's why we are getting the key error like this- `KeyError: "Try running with errors='ignore' as key 'part_types' is not always present"
`
Do you think we can do something else to address this issue? like avoid `part_types` from the response or just ignore the errors from `json_normalize` function